backend/backend.go

```diff
@@ -39,7 +39,7 @@ type Backend interface {
 	PutBucketAcl(_ context.Context, bucket string, data []byte) error
 	DeleteBucket(context.Context, *s3.DeleteBucketInput) error
 	PutBucketVersioning(_ context.Context, bucket string, status types.BucketVersioningStatus) error
-	GetBucketVersioning(_ context.Context, bucket string) (*s3.GetBucketVersioningOutput, error)
+	GetBucketVersioning(_ context.Context, bucket string) (*s3response.GetBucketVersioningOutput, error)
```

Comment:
make this s3response.GetBucketVersioningOutput instead of a pointer to match other s3response types